src/Administration/Resources/app/administration/src/app/component/entity/sw-category-tree-field/index.js

```diff
@@ -154,6 +154,12 @@ Component.register('sw-category-tree-field', {
     },
 
     methods: {
+        canSelectCategory() {
```

Comment:
You can return directly the result instead of a function :)
```
canSelectCategory(category) {
	return ['folder', 'link'].indexOf(category.data.type) === -1;
}	
```